modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/criterion/DLItemSelectorCriterionCreationMenuRestrictionUtil.java

```diff
@@ -74,7 +80,13 @@ protected void activate(BundleContext bundleContext) {
 					"model.class.name");
 
 				if (modelClassName != null) {
-					_propertyServiceReferenceMapper.map(
+					PropertyServiceReferenceMapper
```

Comment:
@GabrielHenriquedev Please move the creation of this variable outside the lambda, just put it in the static block before opening the service tracker map. This way you avoid creating the instance repeatedly.